com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -232,6 +232,10 @@ private void BuildDestinationToTransitionInfoTable()
         private void BuildTransitionStateInfoList()
         {
 #if UNITY_EDITOR
+            if (UnityEditor.EditorApplication.isUpdating)
```

Comment:
This might get removed and placed into a really small PR, but it seems to fix the "domain backup" errors you get during serialization with the more recent  versions of Unity.